src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -30,38 +31,42 @@
 public class BookController {
     private final BookService bookService;
 
-    @GetMapping("/")
+    @GetMapping("")
```

Comment:
```suggestion
    @GetMapping
```
Can you remove it? (check in other palaces)